docs/SERVER.md

```diff
@@ -0,0 +1,311 @@
+# Hosting your own SMP Server
+
+## Overview
+
+SMP server is the relay server used to pass messages in SimpleX network. SimpleX Chat apps have preset servers (for mobile apps these are smp8, smp9 and smp10.simplex.im), but you can easily change app configuration to use other servers.
+
+SimpleX clients only determine which server is used to receive the messages, separately for each contact (or group connection with a group member), and these servers are only temporary, as the delivery address can change.
+
+_Please note_: when you change the servers in the app configuration, it only affects which server will be used for the new contacts, the existing contacts will not automatically move to the new servers, but you can move them manually using BETA feature ["Change receiving address"](../blog/20221108-simplex-chat-v4.2-security-audit-new-website.md#change-your-delivery-address-beta) – it will be automated soon.
+
+## Installation
+
+0. Firtsly, install `smp-server`:
+
+   - Manual deployment:
+     - [Compiling from source](https://github.com/simplex-chat/simplexmq#using-your-distribution)
+     - [Using pre-compiled binaries](https://github.com/simplex-chat/simplexmq#install-binaries)
+
+   - Alternatively, you can deploy `smp-server` using:
+     - [Docker container](https://github.com/simplex-chat/simplexmq#using-docker-1)
+     - [Linode StackScript](https://github.com/simplex-chat/simplexmq#deploy-smp-server-on-linode)
+
+Manual installation require some preparations:
+
+1. Create user and group for `smp-server`:
+
+   ```sh
+   sudo useradd -m smp
+   ```
+
+2. Create necessary directories and assign permissions:
+
+   ```sh
+   sudo mkdir -p /var/opt/simplex /etc/opt/simplex
+   sudo chown smp:smp /var/opt/simplex /etc/opt/simplex
+   ```
+
+3. Allow `smp-server` port in firewall:
+
+   ```sh
+   # For Ubuntu
+   ufw allow 5233
+   ```
+
+4. **Optional** — If you're using distribution with `systemd`, create `/etc/systemd/system/smp-server.service` file with the following content:
+
+   ```sh
+   [Unit]
+   Description=SMP server
+   [Service]
+   User=smp
+   Group=smp
+   Type=simple
+   ExecStart=smp-server start
+   ExecStopPost=/usr/bin/env sh -c '[ -e "/var/opt/simplex/smp-server-store.log" ] && cp "/var/opt/simplex/smp-server-store.log" "/var/opt/simplex/smp-server-store.log.bak"'
+   KillSignal=SIGINT
+   TimeoutStopSec=infinity
+   Restart=always
+   RestartSec=10
+   LimitNOFILE=65535
+   [Install]
+   WantedBy=multi-user.target
+   ```
+
+   And execute `sudo systemctl daemon-reload`.
+
+## Configuration
```

Comment:
this is a bit confusing - this title should describe INI file after the server is already installed and running, also it seems to be lacking the explanations how to download, when to put it etc.

Also this document should include the references to our update and restart scripts (we can add them to simplexmq repo).